libc/test/src/stdlib/rand_test.cpp

```diff
@@ -14,11 +14,18 @@
 #include <stdlib.h>
 
 TEST(LlvmLibcRandTest, UnsetSeed) {
+  static int vals[1000];
```

Comment:
why do we need `static` here?

Reply:
It's a larger array, so putting `static` there places it in the `.bss` section instead of the stack which makes it less likely to explode when run on targets with a smaller stack (AKA the GPU).